CHANGELOG.md

```diff
@@ -7,6 +7,193 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 ### Added
 - `art_decor_check.py` file
+- Greeting GitHub actions
+- 190 new decorations:
```

Comment:
190 or 185 ?

Reply:
@sadrasabouri Would you please correct order in `CHANGELOG.md` ? 
For example  :
1. fancy1
2. fancy2
...
instead of :
1. fancy1
2. fancy10
...